dvc/config.py

```diff
@@ -97,7 +97,7 @@ class RelPath(str):
 }
 LOCAL_COMMON = {
     "type": supported_cache_type,
-    Optional("protected", default=False): Bool,
+    Optional("protected", default=False): Bool,  # obsoleted
```

Comment:
So this one is effectively defunct from now on. Just keeping it in the SCHEMA to not break on older configs.

Reply:
This does not respect explicit `protected = False`, does it?

Reply:
@skshetry It doesn't, as there is no clear use case for it.